index.html

```diff
@@ -4,23 +4,28 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <script src="index.js" defer></script>
     <link rel="stylesheet" href="./css/main.css" />
     <link rel="icon" href="./favicon.ico" />
+    <script src="https://kit.fontawesome.com/d1333a485b.js" crossorigin="anonymous"></script>
     <title>Aziz Mejri - Portfolio</title>
   </head>
   <body>
     <header>
-      <h1 id="logo">Aziz Mejri</h1>
-      <button type="button" class="menu-bars">
-        <i class="fas fa-bars"></i>
-      </button>
-      <nav id="desktop-nav">
-        <ul>
+      <nav id="nav">
+        <div class="logo-menu-container">
+          <h1 class="logo active">Aziz Mejri</h1>
+          <button type="button" class="menu-button">
+            <i class="fas fa-bars hamburger-icon active"></i>
+            <i class="fa-solid fa-xmark x-icon"></i>
+          </button>
+        </div>
+        <ul class="nav-menu">
           <li><a class="nav-link" href="#projects">Portoflio</a></li>
           <li><a class="nav-link" href="#about-me">About</a></li>
```

Comment:
- [OPTIONAL] Please note that you have a typo on the word Portfolio, and consider correcting this to improve the professional image of your project.